src/filesystem_helper.cpp

```diff
@@ -306,6 +306,40 @@ path temp_directory_path()
   return path(temp_path);
 }
 
+path create_temp_directory(const std::string & base_name, const path & parent_path)
+{
+  const auto template_path = base_name + "XXXXXX";
```

Comment:
@emersonknapp 
```suggestion
  const auto template_path = base_name + ".XXXXXX";
```
? Or otherwise, the docblock needs a change.

Reply:
changing the docblock - I don't want to force the `.`